websocket/tests/test_app.py

```diff
@@ -6,6 +6,7 @@
 import websocket as ws
 import ssl
 import unittest
+import rel
```

Comment:
This is to fix the testRunForeverDispatcher() test -- the 'dispatcher="Dispatcher"' previously on line 101 isn't correct.

I'm not sure if lines 102-103 will actually make the test work, I don't know how to add rel to the test dependencies.

Do these tests work in the master branch? As far as I can tell, testRunForeverDispatcher() at least doesn't seem like it could have.